crates/fluvio-cluster/src/cli/shutdown.rs

```diff
@@ -11,17 +12,10 @@ use fluvio_command::CommandExt;
 use crate::render::ProgressRenderer;
 use crate::cli::ClusterCliError;
 use crate::progress::ProgressBarFactory;
-use crate::{ClusterError, UninstallError};
+use crate::{ClusterError, UninstallError, InstallationType};
 
 #[derive(Debug, Parser)]
-pub struct ShutdownOpt {
-    /// shutdown local spu/sc
-    #[arg(long)]
-    local: bool,
-
-    #[arg(long)]
-    no_k8: bool,
-}
+pub struct ShutdownOpt;
```

Comment:
Are the any cases where a manual override of the recorded installation_type might be needed?

Reply:
There should not be any cases when you install one type and want to override it for delete or shutdown.
But, if there is a need for a hack, it can be done by creating a profile `fluvio profile add some_profile localhost:1234 local-k8` with setting another installation type and then execute delete or shutdown.